stack.yaml

```diff
@@ -1,4 +1,4 @@
-resolver: lts-8.12
+resolver: lts-8.13
```

Comment:
LTS 8.12 has some bug in it which forces a recompilation of `happy` every time I write `stack build`. I tried deleting `~/.stack` and `.stack-work` and the bug persisted, and I wasted way too much time waiting for a dozen of packages to build on each change. Then I decided to bump the LTS and the problem was gone, fortunately.